huaweicloud/services/acceptance/identitycenter/data_source_huaweicloud_identitycenter_instance_test.go

```diff
@@ -0,0 +1,48 @@
+package identitycenter
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDatasourceIdentityCenter_basic(t *testing.T) {
+	rName := "data.huaweicloud_identitycenter_instance.test"
+	dc := acceptance.InitDataSourceCheck(rName)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatasourceIdentityCenter_basic(),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckOutput("id_is_not_empty", "true"),
+					resource.TestCheckOutput("identity_store_id_is_not_empty", "true"),
+					resource.TestCheckOutput("urn_is_not_empty", "true"),
```

Comment:
use **resource.TestCheckResourceAttrSet** will be better

Reply:
fixed